### ISSUE TITLE
Typos in COLR description

### ISSUE THREAD

**Issue by lemzwerg** [MicrosoftDocs/typography-issues#861]
```
s/Each layer combines a glyph outline from the 'glyf', CFF or CFF2 table/
  Each layer combines a glyph outline from the 'glyf', 'CFF' or 'CFF2' table/
```

---
#### Dokumentdetails

⚠ *Bearbeiten Sie diesen Abschnitt nicht. Er ist für die Verknüpfung von docs.microsoft.com zum GitHub-Artikel erforderlich.*

* ID: a86069cc-49c7-fd1d-fc35-4c5c80295e0f
* Version Independent ID: a86069cc-49c7-fd1d-fc35-4c5c80295e0f
* Content: [cmap - Character To Glyph Index Mapping Table (OpenType 1.9) - Typography](https://docs.microsoft.com/de-de/typography/opentype/otspec190/cmap)
* Content Source: [typographydocs/opentype/otspec190/cmap.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/opentype/otspec190/cmap.md)
* Product: **typography**
* GitHub Login: @PeterCon
* Microsoft Alias: **alib**

**Comment by PeterConstable** [comment]
Thanks.

The convention now used in the spec is to surround tags with ' if the tag is lowercase or has spaces, but not for tags with four uppercase letters.

**Comment by PeterCon** [comment]
Fixed in beta.

**Comment by PeterCon** [comment]
Addressed in [OpenType 1.9](https://docs.microsoft.com/en-us/typography/opentype/spec/). Closing.
